common/components/BalanceSidebar/EquivalentValues.tsx

```diff
@@ -191,6 +191,10 @@ class EquivalentValues extends React.Component<Props, State> {
           <div className="text-center">
             <h5 style={{ color: 'red' }}>{translate('EQUIV_VALS_TESTNET')}</h5>
           </div>
+        ) : network.hideEquivalentValues ? (
+          <div className="text-center">
+            <h5 style={{ color: 'red' }}>{translate('EQUIV_VALS_UNSUPPORTED_UNIT')}</h5>
```

Comment:
@alepc253 Move this to the colocated .scss file.

Reply:
I'm not sure about this... I added this code replicating how it is solved for `network.isTestnet`, does it make sense to be changed?